ethsigner/signer/azure/src/main/java/tech/pegasys/ethsigner/signer/azure/KeyImporter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.azure;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+
+import com.google.common.collect.Lists;
+import com.google.common.io.BaseEncoding;
+import com.microsoft.azure.keyvault.KeyVaultClientCustom;
+import com.microsoft.azure.keyvault.webkey.JsonWebKey;
+import com.microsoft.azure.keyvault.webkey.JsonWebKeyCurveName;
+import com.microsoft.azure.keyvault.webkey.JsonWebKeyOperation;
+import com.microsoft.azure.keyvault.webkey.JsonWebKeyType;
+import org.web3j.crypto.ECKeyPair;
+
+public class KeyImporter {
```

Comment:
Looks like a test utility maybe?

Reply:
Its not a test util per se - its something which you may need to do in order to create an Azure Key of a specific value .... thought it was useful to have around - should we move it to a test support directory?